src/main/java/com/microsoft/aad/msal4j/ClientCredentialFactory.java

```diff
@@ -10,7 +10,8 @@
 import java.security.cert.X509Certificate;
 
 /**
- * Factory for creating client credentials used in confidential client flows
+ * Factory for creating client credentials used in confidential client flows. For more details, see
+ * https://aka.ms/msal4j-client-credentials
```

Comment:
Can we update this aka.ms link to https://docs.microsoft.com/en-us/azure/active-directory/develop/scenario-daemon-acquire-token?tabs=java#acquiretokenforclient-api

Reply:
The docs link you're pointing to doesn't say anything about ClientCredentialFactory (how to initialize a client credential for confidential clients). The wiki page is more useful.

Reply:
Based on the new patterns in the dev samples, we are going to update docs pages. We can probably update the links once the docs pages are updated.

Reply:
Sounds good, I'm fine with that.